tmb_normalization/code/01-download-tcga-mutect2.py

```diff
@@ -0,0 +1,31 @@
+# This script uses the id from the maf_manifest.tsv and query the GDC API data endpoint for download
+# the downlaoded the files will be in their respective folder - needed to move them out of their own
+# folder and put in `maf_files`
+
+import requests
+import json
+import pandas as pd
+import re
+import os
+
+# read in the manifest as a dataframe and subset the id column
+maf_manifest = pd.read_csv('../results/tcga_not_in_pbta_maf_manifest.tsv', sep='\t')
```

Comment:
would be nice to have a way to add the `.git` root directory and then follow the sub-directories. That way this script can be run from any folder within OMPARE

Reply:
Not important - so I merged this _as is_ and we can revisit in future if needed (unlikely)